math/lcm_sum.cpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * @file
+ * @brief An algorithm to calculate the sum LCM(1,n) + LCM(2,n) + .. + LCM(n,n), where LCM(i,n)
+ * denotes the Least Common Multiple of the integers i and n. For n greater than or equal to 1.
+ * The value of the sum is calculated by formula:
+ * ∑LCM(i, n) = ((∑(d * ETF(d)) + 1) * n) / 2
+ *
+ * @author [Chesta Mittal](https://github.com/chestamittal)
+ */
+
+#include <iostream> /// for std::cin and std::cout
+#include <cassert>  /// for assert
+#include <vector>   /// for std::vector
+
+/**
+ * @namespace math
+ * @brief Mathematical algorithms
+ */
+namespace math {
+    /**
+     * Function to compute sum of euler totients in sumOfEulerTotient vector
+     * @param num input number
+     * @returns int Sum of LCMs, i.e. ∑LCM(i, num) from i = 1 to num
+     */
+    int lcmSum(int num) {
+   
+        int i=0, j=0;
+        int limit = 1000;
```

Comment:
shouldn't the limit be upto the input number?

Reply:
@chestamittal u can remove the `limit` and directly use the `num` you have passed as a parameter to the function. Also, the vector's size should be num+1:
`vector <int> eulerTotient(num + 1);`   
`vector <int> sumOfEulerTotient(num + 1);`

Reply:
@kvedala 
Changed.
@ridhishjain 
Thanks for your suggestion. 😃